Jenkinsfile-utils

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env groovy
```

Comment:
See comment in doc/ file below.

If we are going this route, I am inclined to define an array of all the possible configurations here in this file.
Then there is a single source of truth here.

Reply:
I responded in the doc file but I don't think that is the most elegant solution.